src/CommonDBTM.php

```diff
@@ -6555,6 +6571,14 @@ public static function displayFullPageForItem(
         if (!isset($options['id'])) {
             $options['id'] = $id;
         }
+
+        if (isset($_GET['position'])) {
+            $options['position'] = $_GET['position'];
+        }
+        if (isset($_GET['room'])) {
+            $options['room'] = $_GET['room'];
+        }
```

Comment:
This code was present in at least 3 forms, that's why I decided to standardize it right here

Reply:
This should be controlled by the target class. Indeed, we may want to allow this only for some classes.
Maybe a `public function getFormOptionsFromUrl(array $query_params): array` method could help to do this.

Reply:
Will do